cumulus/parachains/runtimes/assets/asset-hub-rococo/src/xcm_config.rs

```diff
@@ -864,10 +863,7 @@ pub mod bridging {
 					EthereumNetwork::get(),
 					Some(sp_std::vec![Junctions::Here]),
 					SiblingBridgeHub::get(),
-					Some((
-						XcmBridgeHubRouterFeeAssetId::get(),
-						BridgeHubEthereumBaseFee::get(),
-					).into())
```

Comment:
Remove the `BridgeHubEthereumBaseFee` config which is unstable and hard to maintain on chain.